src/roktManager.ts

```diff
@@ -136,6 +138,22 @@ export default class RoktManager {
         }
     }
 
+    public hashAttributes(attributes: IRoktPartnerAttributes): Promise<Record<string, string>> {
+        if (!this.isReady()) {
+            this.queueMessage({
```

Comment:
Since these aren't `void`s like `selectPlacements` is, I'm not sure that from a real life use case perspective this makes sense.  If you try calling `hashAttributes`, before `isReady()` is true, you get a resolved promise, and then the code continues on its way.  _Eventually_, the actual hashed attributes are resolved but can't be used since you're returned something already, right?

Reply:
I see your point.

I'm thinking we may want to revisit the `queueMessage` function, because I would assume that a user would call `hashAttributes` before `selectPlacements`.

Reply:
yea - when the sdk loads on the home page and a user navigates through, it will 100% be available, but if a customer puts a placement on the homepage, then it might not and may necessitate something ilke an mparticle.Rokt.isReady function, which isn't great but may be necessary